man/build_graphical_index.py

```diff
@@ -67,11 +67,10 @@ def main(ext):
         )
 
     with open(os.path.join(man_dir, output_name + f".{ext}"), "w") as output:
-        output.write(
-            header1_tmpl.substitute(
-                title=f"GRASS {grass_version} Reference Manual - Graphical index"
-            )
-        )
+        title = "Graphical index"
+        if ext == "html":
+            title = f"GRASS {grass_version} Reference Manual - {title}"
```

Comment:
Should this be Documentation?

Reply:
I'm not changing it for HTML (as opposed to Markdown/MkDocs), so I'll leave these as is.